osu.Game/Screens/Play/HUD/HoldForMenuButton.cs

```diff
@@ -251,6 +251,7 @@ public bool OnPressed(GlobalAction action)
                 switch (action)
                 {
                     case GlobalAction.Back:
+                    case GlobalAction.PauseGameplay: // in the future this behaviour will differ for replays etc.
```

Comment:
as correctly pointed out by @Joehuu, this same case label should be also added in `OnReleased(GlobalAction)`.